packages/go_router/lib/src/route.dart

```diff
@@ -437,6 +438,9 @@ class GoRoute extends RouteBase {
   /// ```
   final ExitCallback? onExit;
 
+  /// Whether the path is case sensitive or not.
```

Comment:
Can you add more description? perhaps with some example

Reply:
Sure, I added more in [Add better name and documentation](https://github.com/flutter/packages/pull/8992/commits/48ba2860c7edc3143ee53d5590aa35f04b1982d4), tell me if you need more